scripts/install_olm.sh

```diff
@@ -7,13 +7,14 @@
 SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 ROOTDIR="$(dirname "$SCRIPTDIR")"
 DEPLOYDIR="$ROOTDIR/deploy/olm"
-
+VERIFYSCRIPT="verify.sh"
 # Constants
 CATALOGSOURCE="dell-csm-catalogsource"
 OPERATORGROUP="dell-csm-operatorgroup"
 SUBSCRIPTION="dell-csm-subscription"
 COMMUNITY_MANIFEST="operator_community.yaml"
 
+NAMESPACE="test-csm-operator-olm"
```

Comment:
Please pick the namespace from manifest file instead of hardcoding. This can be taken care in the next PR